client/opentracing/src/main/java/io/opentracing/APMSpan.java

```diff
@@ -59,22 +59,22 @@
      * @param builder  The span builder
      * @param recorder The trace recorder
      */
-    public APMSpan(APMSpanBuilder builder, TraceRecorder recorder, ContextSampler sampler) {
+    public APMSpan(APMSpanBuilder builder, TraceRecorder recorder, ContextSampler sampler, String serviceName, String buildStamp) {
```

Comment:
Other approach could be to pass a map representing the additional properties, as it would allow scope for extending provided values in the future - not sure yet what else may be required, but atleast we are not restricting it based on constructor parameters.